scheduler/src/cook/kubernetes/controller.clj

```diff
@@ -27,8 +27,10 @@
 (defn existing-state-equivalent?
   "Is the old and new state equivalent?"
   [old-state new-state]
-  (= old-state new-state) ; TODO. This is not right. We need to tune/tweak this to suppress otherwise identical states so we don't spam.
-  )
+  ; TODO:
+  ; This is not right. We need to tune/tweak this to
+  ; suppress otherwise identical states so we don't spam.
+  (= (dissoc old-state :ancillary) (dissoc new-state :ancillary)))
```

Comment:
we decided we don't want the ancillary field

Reply:
Removed, thanks